app/api/routers/breeze_buddy/merchants/handlers.py

```
@@ -0,0 +1,51 @@
+
+    try:
+        # Get all unique merchants from database
+        merchants = await get_all_merchants()
+
+        logger.info(
+            f"Returning {len(merchants)} merchants to admin {current_user.username}"
+        )
+
+        return MerchantsResponse(merchants=merchants, total=len(merchants))
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)